index.html

```diff
@@ -476,9 +474,10 @@ <h3>
             When a <a>protocol handler description</a>
             <var>protocol_handler</var> having [=manifest=] <var>manifest</var>
             is invoked, it goes through the same steps used to [=invoke a
-            protocol handler=] defined in [=HTML=], where the user agent SHOULD
-            navigate to [=url=] and the appropriate browsing context is set to
-            a new top level browsing context.
+            protocol handler=] where the user agent,
```

Comment:
Hmm this is messy.

Couple of thoughts:

1. Should we also hook into regular HTML protocol handlers so they trigger the launch as well? Or is this a special property you only get if you register using the app manifest? That's a big can of worms (you would need to edit HTML to call the launch with handler algorithm).
2. This existing text is probably wrong, or actually, redundant ([could be converted into non-normative](https://github.com/WICG/manifest-incubations/issues/64)). See above, the processing the manifest actually calls HTML's **register a protocol handler**. That means we should not need normative text to handle a protocol launch, it should just happen via HTML. So this text could just explain that that automatically happens. But that means we can't modify its behaviour to do anything different to plain HTML protocols.
3. Maybe we could just leave it hand-wavy for now.

Reply:
I don't think we want manifest protocol handlers to ever not launch the web app, I wouldn't leave that detail up to user agents to work out.
This section is gnarly at the moment and I think reworking it is out of scope for this change so I opt for 3.